RNAudioPlayerURL.m

```diff
@@ -19,9 +18,14 @@ @implementation RNAudioPlayerURL
   	addObserver:self selector:@selector(playerItemDidReachEnd:) name:AVPlayerItemDidPlayToEndTimeNotification object:self.audioItem];
 }
 
+- (NSArray<NSString *> *)supportedEvents
+{
+  return nil;
+}
+
```

Comment:
weird was working without this yesterday, but without this app crashes (and you can see why).